run_desktop_tests.sh

```diff
@@ -16,7 +16,10 @@
 set -ex
 
 echo "Checking formatting..."
-cargo fmt --all -- --check
+cargo fmt -- --check
+cd libraries/opensk
+cargo +nightly fmt --all -- --check
```

Comment:
nit: We probably also don't need `--all` here.

Same in the rest of the file.

Reply:
Done